tests/test_cesm_fv.py

```diff
@@ -0,0 +1,71 @@
+import shutil
+import warnings
+from pathlib import Path
+
+import pandas as pd
+import pytest
+from filelock import FileLock
+
+from monetio.sat._tropomi_l2_no2_mm import open_dataset
+
+HERE = Path(__file__).parent
+
+
+def retrieve_test_file():
+    fn = "CAM_chem_merra2_FCSD_1deg_QFED_world_201909-01-09_small_sfc.nc"
+
+    # Download to tests/data if not already present
+    p = HERE / "data" / fn
+    if not p.is_file():
+        warnings.warn(f"Downloading test file {fn} for CESM-FV test")
+        import requests
+
+        r = requests.get(
+            "https://csl.noaa.gov/groups/csl4/modeldata/melodies-monet/data/"
+            + f"example_model_data/cesmfv_example/{fn}",
+            stream=True,
+        )
+        r.raise_for_status()
+        with open(p, "wb") as f:
+            f.write(r.content)
+
+    return p
+
+
+@pytest.fixture(scope="session")
```

Comment:
```suggestion
@pytest.fixture(scope="module")
```
Changed this in the others in #183